packages/frontend-vitals-monitoring/README.md

```diff
@@ -0,0 +1,149 @@
+
+   ```sh
+   npm install @govuk-one-login/frontend-vitals-signs
+   ```
+2. Import the package in your Node.js application's startup file (example: app.js or index.js):
+
+   ```js
+   const { frontendVitalsInit } = require("@govuk-one-login/frontend-vital-signs");
+   ```
+
+  [!WARNING] Verify the path to your node module folder is the correct.[!WARNING]
+
+3. In your .env file, add the LOG_LEVEL variable and set it to the desired log level. The   available log levels are:
+
+   - fatal: Logs critical errors that may cause the application to crash.
+   - error: Logs error messages indicating something went wrong.
+   - warn: Logs warnings about potential issues that are not errors but might need attention.
+   - info: Logs informational messages about the application's operation.
+   - debug: Logs debug information useful for development.
+   - trace: Logs detailed information for debugging purposes.
+   - silent: Suppresses all logging output.
+
+  Example:
+
+  Setting the log level to info will log informational messages and errors. To view the logs of our metric object, set it as follows:
+
+  ```dotenv
+  LOG_LEVEL=info
+
+  Example Log:
+
+   ```json
+  {
+    "level": 30,
+    "time": 1722420636051,
+    "pid": 25367,
+    "hostname": "GDS12516",
+    "name": "@govuk-one-login/frontend-vitals-signs",
+    "version": "0.0.1",
+    "requestsPerSecond": {
+      "dynamic": 1,
+      "static": 0.6
+    },
+    "avgStaticResponseTime": 1.5,
+    "avgDynamicResponseTime": 14.5
+  }
+
+   ```
+
+4. Configure the server to use the frontendVitalsInit function. This function initialises the frontend vitals monitoring.
+
+  Example:
+
+   ```js
+
+  const server = app.listen(port, () => {
+    console.log(`Example app listening on port ${port}`);
+  });
+  
+  frontendVitalsInit(server, {
+    interval: 10000, 
+    metrics: ["requestsPerSecond", "avgResponseTime"],
+    staticPaths: ["/assets", "/ga4-assets", "/javascript", "/stylesheets"], 
+  });
+
+   ```
+
+ 
+## `frontendVitalsInit(server, options)`
+
+### Parameters
+
+- **`server`** (required): 
+  The HTTP or HTTPS server instance that you want to monitor.
+
+- **`options`** (optional): 
+  An object containing the following properties:
+  
+  - **`interval`** (optional, number): 
+    The interval in milliseconds for logging metrics. The default value is `10000` (10 seconds).
+
+  - **`metrics`** (optional, array): 
+    An array of strings specifying which metrics to log. Possible values include:
+    - `"requestsPerSecond"`: Logs the number of requests per second.
+    - `"avgResponseTime"`: Logs the average response time.
+    - `"maxConcurrentConnections"`: Logs the maximum number of concurrent connections.
+    - `"eventLoopDelay"`: Logs lag building up due to Node's event loop getting overloaded by asynchronous operations.
+    - `"eventLoopUtilization"`: Logs the time the event loop spends in active and idle states.
+
+  - **`staticPaths`** (optional, array): 
```

Comment:
This looks great, I think it might just be worth adding a bit to say that any strings provided are converted to a regular expression with the format `^{string_value}`, meaning that the path is treated as the starting part of the string, with anything else following the string still matched.

Reply:
added